modules/user_data/variables.tf

```diff
@@ -88,14 +88,24 @@ variable "pg_password" {
   type        = string
 }
 
+# variable "ca_certificate_secret" {
+#   type = object({
+#     arn = string
+#   })
+#   description = <<-EOD
+#   A Secrets Manager secret which contains the Base64 encoded version of a PEM encoded public certificate of a
+#   certificate authority (CA) to be trusted by the EC2 instance.
+#   EOD
+# }
+
 variable "ca_certificate_secret" {
-  type = object({
-    arn = string
-  })
+  default     = null
```

Comment:
We want to avoid adding defaults to any variables of submodules: they can introduce bugs where the root module forgets to pass an argument.